### ISSUE TITLE
XSEDE Application/Software Information

### ISSUE THREAD

**Issue by jpnavarro** [SGCI/sgci-resource-inventory#23]
XSEDE service providers currently publish software module information as shown by the attached file.
[module.txt](https://github.com/SGCI/sgci-resource-inventory/files/7024126/module.txt)

If the SGCI resource inventory teams is interested, XSEDE could propose an additional to the schema for this type of information. Perhaps something to discuss in a future monthly summit. If added to the schema, XSEDE would implement publishing this information in the new version of the schema (v1.1 or whatever).

**Comment by joestubbs** [comment]
I like the idea of working on this issue for 1.1

**Comment by smarru** [comment]
Yes this is a good feature addition and will make sense to for 1.1
